src/Symfony/Component/Notifier/Tests/ChatterTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Symfony\Component\Notifier\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Notifier\Chatter;
+use Symfony\Component\Notifier\Tests\Transport\DummyMessage;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+class ChatterTest extends TestCase
+{
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|TransportInterface
```

Comment:
```suggestion
     * @var \PHPUnit\Framework\MockObject\MockObject&TransportInterface
```
?

And please import the MockObject class

Thanks